functions

```diff
@@ -70,9 +70,14 @@ service_create_container() {
   DATABASE_NAME="$(get_database_name "$SERVICE")"
   docker exec "$SERVICE_NAME" su - postgres -c "createdb -E utf8 $DATABASE_NAME" 2> /dev/null || echo 'Already exists'
 
-  dokku_log_verbose_quiet "Securing connection to database"
-  service_stop "$SERVICE" > /dev/null
-  docker run --rm -i -v "$SERVICE_ROOT/data:/var/lib/postgresql/data" "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION" bash -s < "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/scripts/enable_ssl.sh" &> /dev/null
+
+  service_stop "$SERVICE" > /dev/null;
+  if [[ -z "$DISABLE_SSL" ]]; then
```

Comment:
This doesn't persist at the moment, so if the container is recreated, we'll re-enable ssl.